src/jira-issues-provider.ts

```diff
@@ -59,17 +59,21 @@ export class JiraIssuesProvider implements TreeDataProvider<TreeItem> {
 				.then(async function(data: any){
 					var children: JiraIssue[] = [];
 					var promises = data.issues.map( async(issue: any) => {
-						const jiraIssue = new JiraIssue(`${issue.key}: ${issue.fields.summary}`, issue);
-						const issueType = issue.fields.issuetype.name;
-						var icon = `${issueType.toLowerCase()}.svg`;
+						var description = `${issue.key}: (${issue.fields.status.name}) ${issue.fields.summary}`
+						const jiraIssue = new JiraIssue(description, issue);
+						if (issue.fields.status.name == "Done") {
+							var icon = `task_complete.svg`;
```

Comment:
It might be a bit confusing to have all issues types bugs, epics, stories look visually like a task when they're complete.
If we want to add a visual distinction for "done" tasks (which I'm in favor of) I think all issue types should have a custom "done" svg. For example, it might be confusing to see an bug that is complete visually appear the same as a task that is complete.

Reply:
Hmmmm. I'm not the best graphic designer. Do you have any suggestions on how to whip up some graphics?

Reply:
I was just thinking we could color them in green like you did for the task_completed. I think we can just update the svg code to do that. I'll take a peek tonight, also not the best at design.

Reply:
I can give that a shot.